src/main/kotlin/org/jitsi/nlj/util/ClockUtils.kt

```diff
@@ -28,3 +28,11 @@ val NEVER: Instant = Instant.MIN
 fun Instant.formatMilli(): String = TimeUtils.formatTimeAsFullMillis(this.epochSecond, this.nano)
 
 fun Duration.formatMilli(): String = TimeUtils.formatTimeAsFullMillis(this.seconds, this.nano)
+
+fun <T> Iterable<T>.sumOf(selector: (T) -> Duration): Duration {
+    var sum: Duration = Duration.ZERO
+    for (element in this) {
+        sum += selector(element)
```

Comment:
I had thought that kotlin also overrode `minus` for `Duration` to do `Duration.between` (which you have to do in a few other places manually), like here it supports add, but I played with it a bit and it doesn't look like it works.  Weird.

Reply:
I think you can use `minus` for subtracting `Duration`s, but not for subtracting `Instant`s.